src/plugins/agentDesktop/agentdesktop.ts

```diff
@@ -216,9 +217,12 @@ class AgentDesktopPlugin {
     removeEmptyBubblesInTemplate() {
         let me: any = this;
         let cwInstance = me.hostInstance;
+        console.log('in removeEmptyBubblesInTemplate fn',cwInstance);
         class customTemplateComponent {
             renderMessage(msgData: any) {
-                if (msgData?.type === "currentUser" && msgData?.message[0]?.cInfo?.body === "") {
+                console.log('msgData',msgData);
+                if (msgData?.type === "currentUser" && msgData?.message[0]?.cInfo?.body === "" && !msgData.message.body) {
```

Comment:
@mohithape-kore It should be a OR condition. msgData?.message[0]?.cInfo?.body === "" || !msgData.message.body.

also did you check if !msgData.message.body is valid?

Please remove console logs. as it sits in the customer websites. its not a good idea to expose these console.logs.

Reply:
@KambagiriswamyBo-Kore as per comments changed the code. can you validate once.